# type: "effort_controllers/JointGroupPositionController"
# joints: 
#  - base_link_armLink0_joint
#  - armLink0_armLink1_joint
#  - armLink1_armLink2_joint
#  - armLink2_armLink3_joint

joint0_position_controller:
    type: position_controllers/JointPositionController
    joint: base_link_armLink0_joint   
joint1_position_controller:
    type: position_controllers/JointPositionController
    joint: armLink0_armLink1_joint
joint2_position_controller:
    type: position_controllers/JointPositionController
    joint: armLink1_armLink2_joint
joint3_position_controller:
    type: position_controllers/JointPositionController
    joint: armLink2_armLink3_joint

gazebo_ros_control:
  pid_gains:
    base_link_armLink0_joint: {p: 10.0, i: 0.001, d: 0.01}
    armLink0_armLink1_joint: {p: 10.0, i: 0.001, d: 0.01}
    armLink1_armLink2_joint: {p: 10.0, i: 0.001, d: 0.01}
    armLink2_armLink3_joint: {p: 10.0, i: 0.001, d: 0.01}

